core/src/main/resources/hudson/ProxyConfiguration/config.groovy

```diff
@@ -11,8 +11,8 @@ f.entry(title:_("Port"),field:"port") {
 f.entry(title:_("User name"),field:"userName") {
     f.textbox()
 }
-f.entry(title:_("Password"),field:"password") {
-    f.password(value:instance?.encryptedPassword)
+f.entry(title:_("Password"),field:"secretPassword") {
+    f.password(value:instance?.secretPassword)
```

Comment:
Now that `secretPassword` is used both in `value` and `field`, this could be simplified as `f.password()` - no?

Reply:
correct, done